src/components/Callout/Callout.tsx

```diff
@@ -29,14 +28,18 @@ export class Callout extends React.Component<ICalloutProps, ICalloutState> {
   private _hostElement: HTMLDivElement;
   private _calloutElement: HTMLDivElement;
   private _events: EventGroup;
+  // There are some cases when the callout needs to be rendered on a window other than the one
+  // the javascript is running in.
+  // NOTE: The window must be within the same domain as the parent window or this will not be allowed.
+  private _targetWindow: Window;
 
   constructor(props: ICalloutProps) {
     super(props);
-
+    this._targetWindow = props.targetElement ? props.targetElement.ownerDocument.defaultView : window;
```

Comment:
I think caching this seems like a bad idea. If my targetElement were to change... maybe this decision would need to be re-evaluated. Why not just resolve this inside the render?